fastlane/Fastfile

```diff
@@ -59,6 +59,94 @@ lane :github_release do |options|
   )
 end
 
+desc "Update dependencies to latest GitHub releases"
+lane :update_native_dependencies_to_latest do |options|
+  latest_ios_release_info = get_latest_github_release(url: "RevenueCat/purchases-ios")
```

Comment:
Hmm more than a `url` this seems more a `repo` or `repo_name`... What do you think?